src/parser/jobs/ast/modules/LucidDreaming.js

```diff
@@ -108,85 +66,27 @@ export default class LucidDreaming extends Module {
 	}
 
 	_onComplete() {
-		// console.log(this)
 		//uses missed reported in 1 decimal
 		const holdDuration = this._uses === 0 ? this.parser.fightDuration : this._totalHeld
-		// console.log(holdDuration)
-		// console.log(this.parser.formatDuration(holdDuration))
 		const _usesMissed = Math.floor(holdDuration / (ACTIONS.LUCID_DREAMING.cooldown * 1000))
-		// console.log('no mp: ' + this._gcdCountHoldingLucid)
-		const warnTarget = (this._uses - 1) / this._uses * 100
-		const failTarget = (this._uses - 2) / this._uses * 100
-
-		this.checklist.add(new TieredRule({
-			name: <Trans id="ast.lucid-dreaming.checklist.name">
-				Extend Lucid Dreaming
-			</Trans>,
-			description: <Trans id="ast.lucid-dreaming.checklist.content">
-				Astrologians have a very low MP pool, due to the high cost of their healing spells. If they adhere to "Always be casting" they frequently
-				find themselves desiring more MP. It's important to extend all casts of Lucid Dreaming with <ActionLink {...ACTIONS.CELESTIAL_OPPOSITION} /> to maximize MP benefits.
-			</Trans>,
-			tiers: {[warnTarget]: TARGET.WARN, [failTarget]: TARGET.FAIL, 100: TARGET.SUCCESS},
-			requirements: [
-				new Requirement({
-					name: <Trans id="ast.lucid-dreaming.checklist.requirement.name">
-						<ActionLink {...ACTIONS.LUCID_DREAMING} /> extensions
-					</Trans>,
-					value: this._extensions,
-					target: this._uses,
-				}),
-			],
-		}))
 
 		if (_usesMissed > 1 || this._uses === 0) {
 			this.suggestions.add(new Suggestion({
 				icon: ACTIONS.LUCID_DREAMING.icon,
 				content: <Fragment>
 					<Trans id="ast.lucid-dreaming.suggestion.content">
-					Keep <ActionLink {...ACTIONS.LUCID_DREAMING} /> on cooldown for better MP management, unless there's a specific part of the fight you need to drop aggro quick.
+					Keep <ActionLink {...ACTIONS.LUCID_DREAMING} /> on cooldown for better MP management.
```

Comment:
Seems a bit terse, would be good to flesh this out a bit... but I'm not really sure what there is to add, hah.